pkg/cmd/cli/backup/describe.go

```diff
@@ -17,18 +17,55 @@ limitations under the License.
 package backup
 
 import (
+	"fmt"
+
 	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
+	"github.com/heptio/ark/pkg/apis/ark/v1"
 	"github.com/heptio/ark/pkg/client"
+	"github.com/heptio/ark/pkg/cmd"
+	"github.com/heptio/ark/pkg/cmd/util/output"
 )
 
-func NewDescribeCommand(f client.Factory) *cobra.Command {
+func NewDescribeCommand(f client.Factory, use string) *cobra.Command {
+	var listOptions metav1.ListOptions
+
 	c := &cobra.Command{
-		Use:   "describe",
-		Short: "Describe a backup",
+		Use:   use,
```

Comment:
the help text shows this as `ark backup describe [flags]` -- would be nice to add a `[NAME1] [NAME2] ...` before flags so it's clear can optionally take one/several. Ditto for `restore` and `schedule`.

Reply:
Updated, PTAL